>From c633291e4d8ad472f1f6a04749cb41f41b284621 Mon Sep 17 00:00:00 2001
From: Didier Raboud <odyx@debian.org>
Date: Thu, 3 Dec 2015 20:53:26 +0100
Subject: [PATCH] Replace ries.d.o with mirror.ftp-master.debian.org alias for
 the developer accessible archive copy

---
 common.ent | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common.ent b/common.ent
index a1ec2b0..62b93d8 100644
--- a/common.ent
+++ b/common.ent
@@ -51,7 +51,7 @@
 <!ENTITY ftp-upload-host "ftp.upload.debian.org">
 <!ENTITY ftp-eu-upload-host "ftp.eu.upload.debian.org">
 <!ENTITY ssh-upload-host "ssh.upload.debian.org">
-<!ENTITY ftp-master-mirror "ries.debian.org">
+<!ENTITY ftp-master-mirror "mirror.ftp-master.debian.org">
 <!ENTITY upload-queue "/pub/UploadQueue/">
 
 <!ENTITY url-debian-policy "https://&www-debian-org;/doc/debian-policy/">
-- 
2.6.2

